### ISSUE TITLE
Voice models/training 

### ISSUE THREAD

**Issue by JuniorGamingTime** [myshell-ai/MeloTTS#95]
So I have MeloTTS running on a docker faster than real time (GPU) and a FastAPI wrapper. But was wondering are there any other pre train models to use? I want to be able to change the voice to a male. 

Also for training my own model how much data do I need? And when getting the training data should only include the voice of how I want the end model to sound like?

Never done anything like this, so I bit lost!

**Comment by Swastik-Mantry** [comment]
Hey @louistiti, Could you please share the time taken / ETA for fine-tuning/training the model ?
Fine-tuning is going on for couple of days, can't understand whats the issue. Thanks in advance.

**Comment by louistiti** [comment]
> Hey @louistiti, Could you please share the time taken / ETA for fine-tuning/training the model along with your hardware details ? Fine-tuning is going on for couple of days, can't understand whats the issue. Thanks in advance.

Sure! I got a decent quality after 36 hours of training already. It took ~3.5 days for the training to be complete.

**Comment by s-tweed** [comment]
Sounds pretty impressive! What is your GPU setup? I want to fine-tune a custom voice but 3.5 days sounds like quite a while...

**Comment by louistiti** [comment]
> Sounds pretty impressive! What is your GPU setup? I want to fine-tune a custom voice but 3.5 days sounds like quite a while...

I have an RTX 3090.

**Comment by s-tweed** [comment]
Thanks for the info, gives me a point of reference. I really like the voice you trained, it's quite fun to hear. I hope to see your project continue to advance!

**Comment by FabioAML** [comment]
Hi @louistiti , have you modified the training.py ? for me it keeps looping after finishing the epochs.

**Comment by louistiti** [comment]
> Hi @louistiti , have you modified the training.py ? for me it keeps looping after finishing the epochs.  

I only modified the config file to set the batch size to 8 according to my hardware. I also noticed that once all iterations are over it'll loop indefinitely so I just stop it manually 😅

**Comment by FabioAML** [comment]
hahaha thanks. I was so desperate trying to find the loop that didn't think for a second about stopping manually.

**Comment by julia-imlauer** [comment]
Hey @louistiti, the trained voice after 36h sounds really impressive. I am trying to train also another voice. However, when I follow the script, the generator loss doesn't seem to decrease at all.

- Which audio files did you use (sample rate, channel)?
- Did you select one of the English models for fine tuning (if yes, how did you do that)?
- Can you maybe share parts of your meta.txt.cleaned and your config.json?

I would really appreciate your help!
